### PR TITLE
Multi-target-licking behavior ingestion 

### PR DESCRIPTION
**Pull request by ttngu207**
Added tables related to multi-target-licking task: `MultiTargetLickingSessionBlock`
Add new lookup values to `Task` - requires changing table attribute: `task` to `varchar(24)`

Table definition updates to:
Task
TaskProtocol
SessionTask
BehaviorTrial

### CODE REVIEW

File: pipeline/experiment.py [hunk 1]
```diff
@@ -69,7 +72,10 @@ class WaterPort(dj.Lookup):
     water_port: varchar(16)  # e.g. left, right, middle, top-left, purple
```
Comment (ixcat) [inline pipeline/experiment.py:72]: might make sense to comment-document the 'mtl' acronym as being related to the multi-target-licking experiment, and perhaps a tiny bit about the 1-9 target arrangement (guessing a 'number pad' like 3x3 based on the 1-9 range?). could also go to the right of the actual contents zip, or refer people to the WaterPort part table which seems to have a bit more details

File: pipeline/experiment.py [hunk 2]
```diff
@@ -150,7 +156,8 @@ class TrialNoteType(dj.Lookup):
     definition = """
     trial_note_type : varchar(24)
     """
-    contents = zip(('autolearn', 'protocol #', 'bad', 'bitcode', 'autowater', 'random_seed_start'))
+    contents = zip(('autolearn', 'protocol #', 'bad',
+                    'bitcode', 'autowater', 'random_seed_start'))
```
Comment (ixcat) [inline pipeline/experiment.py:160]: similar to the 'mtl-' comment, might make sense to document which values go with which experiment in comments

**Review by ixcat — APPROVED** [review]
comments mostly nits; review wise, looks clean generally, but as a big change probably should be sure to regression test against delayed response and foraging as well as verify the new multi-target licking if it hasn't already been done.

File: pipeline/experiment.py [hunk 3]
```diff
@@ -59,7 +60,9 @@ class TaskProtocol(dj.Lookup):
         ('s1 stim', 8, 'mini-distractors and full distractors (only at late delay), with different levels of the mini-stim and the full-stim during sample period'),
         ('s1 stim', 9, 'mini-distractors and full distractors (only at late delay), with different levels of the mini-stim and the full-stim during sample period'),
         ('foraging', 100, 'moving lickports, delay period, early lick punishment, sound GO cue then free choice'),
-        ('foraging 3lp', 101, 'moving lickports, delay period, early lick punishment, sound GO cue then free choice from three lickports')
+        ('foraging 3lp', 101, 'moving lickports, delay period, early lick punishment, sound GO cue then free choice from three lickports'),
+        ('multi-target-licking', 1, 'multi-target-licking task - 2D'),
+        ('multi-target-licking', 2, 'multi-target-licking task - Spontaneous')
```
[the remark this answers is not in the document]
Comment (ttngu207) [reply pipeline/experiment.py:65]: I will confirm with Dave again to be sure. But these `1` and `2` are how they number the protocol, stored in the behavior .mat file

File: pipeline/experiment.py (hunk 1 above)
Comment (ttngu207) [reply pipeline/experiment.py:72]: good point, commit added to add more comments here

File: pipeline/experiment.py (hunk 2 above)
Comment (ttngu207) [reply pipeline/experiment.py:160]: technically all of these note types are applicable to all 3 types of experiment

Comment (ixcat) [reply pipeline/experiment.py:160]: thanks, yes wasn't sure here. thanks for clarifying, misread the reformat as addition. in that case default is good. :+1:

**Pull request merged**